vizro-core/examples/_chart-gallery/pages/examples/funnel.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+
+funnel_data = pd.DataFrame(
+    {"Stage": ["Leads", "Sales calls", "Follow-up", "Conversion", "Sales"], "Value": [5, 4, 3, 2, 1]}
```

Comment:
I can change the numbers to your suggested numbers, but they will always look like a pyramid shape.

Even if you do something like this = [10, 15, 4, 2, 1], which isn't funnel-like, it will look like a pyramid. The areas will get a bit more space, but the shape itself will always stay like a pyramid.

![Screenshot 2024-08-20 at 18 22 45](https://github.com/user-attachments/assets/c6e6b6e2-cd57-47a7-a002-c9b14d3b5d50)

Reply:
Ooh ok, that makes sense actually. In that case just go for whatever you prefer.